include/random_init_board.h

```diff
@@ -5,6 +5,6 @@
 #include <time.h>
 
 void random_init_board(int *board, int rows, int cols);
-/* Initialise the board with all cells at 0 */
+/* Initialise the board with a random dead or alive sate */

```

Comment:
Sate?

Reply:
Pls keep description above the func

Reply:
The comment part will be handled here along with other functions.
https://github.com/ickule/yagolic/issues/29